cdap-docs/developers-manual/source/data-exploration/custom-datasets.rst

```diff
@@ -302,8 +302,8 @@ When creating your queries, keep these limitations in mind:
 - The query syntax of CDAP is a subset of the variant of SQL that was first defined by Apache Hive.
 - The SQL commands ``UPDATE`` and ``DELETE`` are not allowed on CDAP Datasets.
 - When addressing your datasets in queries, you need to prefix the data set name with the CDAP
-  namespace ``cdap_user_``. For example, if your Dataset is named ``ProductCatalog``, then the corresponding table
-  name is ``cdap_user_productcatalog``. Note that the table name is lower-case.
+  namespace ``dataset_``. For example, if your Dataset is named ``ProductCatalog``, then the corresponding table
```

Comment:
how about leaving out CDAP namespace and just saying "prefix the data set name with dataset_"?  Use of "namespace" may be confusing